aws/modules/cloudwatchslack/module.ftl

```diff
@@ -102,48 +102,41 @@
                 tier : {
                     "Components" : {
                         topicName : {
-                            "topic" : {
-                                "Instances" : {
-                                    "default" : {
-                                        "DeploymentUnits" : [ topicName ]
-                                    }
-                                },
-                                "Subscriptions" : {
-                                    "slack" : {
-                                        "Links" : {
-                                            "lambda" : {
-                                                "Tier" : tier,
-                                                "Component" : lambdaName,
-                                                "Function" : "send"
-                                            }
+                            "Type" : "topic",
+                            "deployment:Unit" : topicName,
+                            "deployment:Priority" : 50,
+                            "Subscriptions" : {
+                                "slack" : {
+                                    "Links" : {
+                                        "lambda" : {
+                                            "Tier" : tier,
+                                            "Component" : lambdaName,
+                                            "Function" : "send"
                                         }
                                     }
                                 }
                             }
                         },
                         lambdaName : {
                             "lambda" : {
```

Comment:
Given the explicit use of the `attribute`, I think this level in the config can be removed?

Reply:
Where do you mean attribute? couldn't see it mentioned

Reply:
Line 121 is the type based attribute "lambda", line 122 is the type attribute. I noticed elsewhere you'd dropped the type attribute (e.g. "topic") so for consistency we should drop this one.

Reply:
Ah right! sorry missed that will fix them up